crates/tauri/src/manager/webview.rs

```diff
@@ -60,7 +60,7 @@ pub(crate) struct IpcJavascript<'a> {
 pub struct UriSchemeProtocol<R: Runtime> {
   /// Handler for protocol
   #[allow(clippy::type_complexity)]
-  pub protocol:
+  pub handler:
```

Comment:
I'm not exactly sure why this had been changed to `protocol` in #1617 as it clearly says `Handler for protocol` in the comment and that's how we call other things as well